pegasus/helpers/form_helpers.rb

```diff
@@ -79,6 +79,10 @@ def insert_or_upsert_form(kind, data, options={})
   row
 end
 
+# @param [String] kind The kind of the form to be updated.
+# @param [String] secret The secret associated with the form to be updated.
+# @param [Hash] data The data to update.
+# @return [Hash] The hash of values sent to the DB for updating.
```

Comment:
:( I've been using the `@param` format in this first [example](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md): `@param name [type]`.

Seeing this, I checked the docs again, and it turns out thir example is misleading. All their other examples use the [default factory](http://www.rubydoc.info/gems/yard/file/docs/TagsArch.md#DefaultFactory) tag format: `@param [type] name`, which I find less readable, but apparently I've been doing it wrong.